tests/PromiseTest/PromiseFulfilledTestTrait.php

```diff
@@ -261,7 +261,7 @@ public function finallyShouldNotSuppressValueWhenHandlerReturnsANonPromiseForFul
 
         $adapter->resolve($value);
         $adapter->promise()
-            ->finally(function () {
+            ->finally(function () { /** @phpstan-ignore-line  */
```

Comment:
These, and all the other ignores on finally functions are there because it's part of the test to break the expected contract.